### PR TITLE
Allow string that has delimiter on testing access.

### PR DESCRIPTION
**Pull request by lloricode**
Allow on test class to set like this

```php

// App\Containers\xx\UI\API\Tests\Functional\GetAllxxxFunctionalTest
    /**
     * Define which Roles and/or Permissions has access to this request.
     *
     * @var  array
     */
    protected $access = [
        'permissions' => 'list-data',
        'roles'       => 'admin|employee',
    ];
```

just like on main `Request`

Thanks :)

### CODE REVIEW

File: Traits/TestsTraits/PhpUnit/TestsAuthHelperTrait.php [hunk 1]
```diff
@@ -187,8 +187,12 @@ private function setupTestingUserAccess($user, $access = null)
     private function setupTestingUserRoles($user, $access)
     {
         if (isset($access['roles']) && !empty($access['roles'])) {
-            if (!$user->hasRole($access['roles'])) {
-                $user->assignRole($access['roles']);
+
+            $roles = is_array($access['roles']) ? $access['roles'] :
+            explode('|', $access['roles']);
```
Comment (johannesschobel) [inline Traits/TestsTraits/PhpUnit/TestsAuthHelperTrait.php]: please don't do this! - it took me 3min to figure out, that L191 and L192 is *one* single command, which is really bad formatted.. I thought you would `explode()` and never use the result... Same on the other function!

File: Traits/TestsTraits/PhpUnit/TestsAuthHelperTrait.php [hunk 2]
```diff
@@ -205,7 +209,11 @@ private function setupTestingUserRoles($user, $access)
     private function setupTestingUserPermissions($user, $access)
     {
         if (isset($access['permissions']) && !empty($access['permissions'])) {
-            $user->givePermissionTo($access['permissions']);
+
+            $permissions = is_array($access['permissions']) ? $access['permissions'] :
+            explode('|', $access['permissions']);
```
Comment (johannesschobel) [inline Traits/TestsTraits/PhpUnit/TestsAuthHelperTrait.php]: same here

File: Traits/TestsTraits/PhpUnit/TestsAuthHelperTrait.php (hunk 1 above)
Comment (lloricode) [reply Traits/TestsTraits/PhpUnit/TestsAuthHelperTrait.php]: hmm ive just follow this https://github.com/apiato/core/blob/master/Abstracts/Requests/Request.php#L259

Comment (johannesschobel) [reply Traits/TestsTraits/PhpUnit/TestsAuthHelperTrait.php]: the code you quoted, however, uses whitespace / tabs to enhance readability.. you did not..
i.e., the `explode` is on the same column as `$roles =`.. Thats what i meant..

Comment (lloricode) [reply Traits/TestsTraits/PhpUnit/TestsAuthHelperTrait.php]: I get it,

Comment (lloricode) [reply Traits/TestsTraits/PhpUnit/TestsAuthHelperTrait.php]: it is ok? I just refactor it

**Pull request closed without merging**